src/components/Head.jsx

```diff
@@ -22,7 +22,7 @@ const Head = ({
     ]}
     <style amp-boilerplate="">{boilerplateStyles}</style>
     <noscript><style amp-boilerplate="">{boilerplateStylesNoScript}</style></noscript>
-    {styles ? <style amp-custom="">{styles}</style> : null }
+    {styles ? <style amp-custom="" dangerouslySetInnerHTML={{__html:styles}}></style> : null }
```

Comment:
Please make use of innerHTML module from utils folder.

```javascript
import innerHTML from '../utils/innerHTML'

{styles ? <style amp-custom="" {...innerHTML(styles)} /> : null }
```

Thanks, i'll add unit test for this.